pycontrails/models/cocip/cocip.py

```diff
@@ -2185,6 +2185,12 @@ def calc_timestep_contrail_evolution(
         },
         copy=False,
     )
+    intersection = contrail_2.coords_intersect_met(met)
+    if not np.any(intersection):
+        warnings.warn(
```

Comment:
is this something that the developer implementing pycontrails can manage/control? e.g. can they do something differently with their met data and problem definition/configuration as to avoid having a contrail advect out of bounds?

Or is this more of a runtime and circumstantial issue, where we simply want to let the developer (and any consumers of the outputs) know that these data may be troubled due to a truncated evolution (but the implementer has no agency over remediating the issue)?

If the latter, I wonder if this warrants a python logged warning, rather than python native (stderr) warning?

Reply:
I think this is more along the lines of the first. If the contrail advects of out of bounds, the developer ideally should have used more met data in running the model.

Occasionally the developer could have done everything correct and still the contrails blows away ... so it's a little debatable here. But I do think the essence of this warning is that the developer could have done something slightly better / different.